<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configura Servidor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>59</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lb_root_dir">
       <property name="text">
        <string>Diret√≥rio Raiz:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_root_dir"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lb_port">
       <property name="text">
        <string>Porta :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txt_port">
       <property name="inputMask">
        <string>00009; </string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lb_speed_limit">
       <property name="text">
        <string>Limite de Vel. :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="txt_speed_limit">
         <property name="inputMask">
          <string>000000000; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_speed_limit">
         <item>
          <property name="text">
           <string>b/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mb/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gb/s</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>296</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="bt_apply">
       <property name="text">
        <string>Aplicar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_cancel">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_exit">
       <property name="text">
        <string>Sair</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
